kitchen-tests/cookbooks/rspec/metadata.rb

```diff
@@ -0,0 +1,8 @@
+name             "rspec"
+license          "Apache-2.0"
+description      "Runs nothing"
```

Comment:
A description of what we're trying to do with this will be useful for the future folks